bundles/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/ModbusGenericBindingProvider.java

```diff
@@ -152,14 +153,19 @@ State getItemState() {
             return item.getState();
         }
 
+        public void setItemState(State state) {
+            ((GenericItem) item).setState(state);
+            ;
```

Comment:
Of course, that would be an easy solution. Just to be sure: As I learned that keeping a reference to an Item is a bad idea, is keeping a reference to a State really save, or could that also potentially cause problems?